provisioning/containers/gui-common/README.md

```diff
@@ -0,0 +1,22 @@
+# Docker-based GUI
+
+This sub-project aims to provide a Dockerized environment where Docker-based GUI applications can run. This is backed by a set of different containers, each with its own specific functions, that together constitute and offer the Docker-based GUI application.
+
+## General architecture
+
+In order to deploy GUI applications in CrownLabs the following is required:
+
+- A VNC-based X server
+- A window manager (it's better to have it since sub-windows or menu opened by the application might not get rendered correctly)
+- Websockify
+- noVNC client files (static HTML files hosted somewhere)
+- the GUI application
+
+## Current implementation
+
+Currently the following containers are used to build the overall GUI application:
+
+- Xvnc server (based on TigerVNC) + fluxbox window manager, which are responsible to provide the basic graphical desktop environment.
+- Websockify (C version), exposing VNC server as a web server
```

Comment:
Document (maybe not here, but it has to be said) why you're using the C version.